### PR TITLE
Global Font Update to Barlow

### PR DESCRIPTION
**Pull request by rdoinhs**
### Description
This PR integrates the Google font "Barlow" along with its two condensed variations via the built-in Next font functionality. References to "Rubik" were removed. A separate font file is introduced as utility function to export/import where needed. This approach ensures condensed variations are preloaded only when rendered in accordance with [https://nextjs.org/docs/app/building-your-application/optimizing/fonts](url).

### CODE REVIEW

**Review by cpilande — COMMENTED** [review]
Looks good! Just to check, are we switching from the B612 font in the Figma wireframe?

**Comment by rdoinhs** [comment]
Oh right, yes, I used Barlow medium for the body text and intend to use bold for headings, so we can for sure update the wireframes.

**Review by cpilande — APPROVED** [review]
Sounds good! Maybe we can check if it would be nice or not to have a different font for the itinerary specifically, so I'll take a look at that in my issue.

File: src/components/Input/input.module.css
Comment (heyitsmass) [inline src/components/Input/input.module.css:1]: Try to leave this out of the pr's if they're only changing formatting so we can avoid bombing important commit blames, Any change comments made by the previous author will be overwritten by this commit.

**Review by heyitsmass — COMMENTED** [review]
[#](https://github.com/heyitsmass/BayView/pull/55/files#r1373337068)

File: src/app/layout.tsx [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
+import { barlow } from "./fonts";
 
-import OAuthProvider from './GoogleProvider';
+import OAuthProvider from "./GoogleProvider";
 
 export default async function Layout({ children }: { children: ReactNode }) {
-  return (
-    <html>
-      <body>
-        <OAuthProvider clientId={process.env.GOOGLE_CLIENT_ID!}>
```
Comment (heyitsmass) [inline src/app/layout.tsx:12]: This change will also be overwritten

**Review by heyitsmass — APPROVED** [review]

File: src/components/Input/input.module.css
Comment (rdoinhs) [reply src/components/Input/input.module.css:1]: Noted and very true.

**Pull request merged**